src/policy_evaluator.rs

```diff
@@ -228,40 +319,53 @@ impl PolicyEvaluator {
             }
         };
 
-        match self
-            .wapc_host
-            .call("validate_settings", settings_str.as_bytes())
-        {
-            Ok(res) => {
-                let vr: Result<SettingsValidationResponse> = serde_json::from_slice(&res)
-                    .map_err(|e| anyhow!("cannot convert response: {:?}", e));
-                vr.unwrap_or_else(|e| SettingsValidationResponse {
-                    valid: false,
-                    message: Some(format!("error: {:?}", e)),
-                })
+        match &self.runtime {
+            Runtime::Wapc(wapc_host) => {
```

Comment:
Same here. I would get rid of this `switch` by defining a common trait. The waPC runtime would perform the usual validation of the settings, the Burrego one would always return `true`.

Reply:
Resolving. https://github.com/kubewarden/policy-evaluator/issues/24 is filed and we can continue over there.